pypeit_files/keck_lris_red_long_400_8500_longread.pypeit

```diff
@@ -0,0 +1,37 @@
+# Auto-generated PypeIt file
+# Wed 19 Aug 2020 20:12:49
+
+# User-defined execution parameters
+[rdx]
+spectrograph = keck_lris_red
+  detnum = 2
+[baseprocess]
+    use_biasimage = False
+    use_illumflat = False
+
+# Setup
+setup read
+ Setup A:
+   --:
+     binning: 1,1
+     dichroic: '560'
+     disperser:
+       angle: 23.33069992
+       name: 400/8500
+     slit:
+       decker: long_1.0
+       slitlen: none
+       slitwid: none
+setup end
+
+# Read in the data
+data read
+ path ../rawdir
```

Comment:
At some point I went through all these pypeit files and made these full paths.  I'm not sure if that's necessary, but I just wanted to check that this works with pypeit_test to reset to the user's local path.

Reply:
yeah, it does seem to work.